core/templates/dev/head/pages/preferences/preferences.html

```diff
@@ -155,19 +155,29 @@
             <div class="col-lg-10 col-md-10 col-sm-10">
               <div class="checkbox">
                 <label>
-                  <input type="checkbox" ng-model="canReceiveEmailUpdates" ng-change="saveEmailPreferences(canReceiveEmailUpdates, canReceiveEditorRoleEmail, canReceiveFeedbackMessageEmail)">
+                  <input type="checkbox" ng-model="canReceiveEmailUpdates" ng-change="saveEmailPreferences(canReceiveEmailUpdates, canReceiveEditorRoleEmail, canReceiveFeedbackMessageEmail,
+                  canReceiveSubscriptionEmail)">
```

Comment:
Just to double-check, you tested this end-to-end, right? I'm not sure whether Angular can handle newlines in the middle of a function call.

That said, it might be a good idea to put this on the end of the previous line anyway. Ditto below.